File: Arrays/Python/RemoveDuplicates.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n = len(nums)
        num = sorted(list(set(nums)))
        res = len(num)
        for i in range(len(nums)):
            if i<res:
                nums[i] = num[i]
            else:
                nums[i] = '_'
        return res

S = Solution()
S.removeDuplicates([1,1,2])

File: Arrays/C++/RemoveDuplicates.cpp
#include <iostream>
#include <vector>

using namespace std;

class Solution
{
public:
    int removeDuplicates(vector<int> &nums)
    {
        int n = nums.size();
        if (n == 0)
        {
            return 0;
        }
        vector<int> v;
        v.push_back(nums[0]);
        for (int i = 1; i < nums.size(); i++)
        {
            if (nums[i - 1] == nums[i])
            {
                continue;
            }
            v.push_back(nums[i]);
        }
        int res_size = v.size();
        for (int j = 0; j < nums.size(); j++)
        {
            if (j < res_size)
            {
                nums[j] = v[j];
            }
            else
            {
                nums[j] = -101;
            }
        }
        return res_size;
    }
};
